bounties_api/std_bounties/client_helpers.py

```diff
@@ -172,17 +173,29 @@ def map_user_data(data_hash, public_address):
     plucked_data = pluck(data, user_data_keys)
     plucked_social = pluck(data.get('social', {}), user_social_keys)
 
+    profileDirectoryHash = data.get('profilePhoto', {}).get('fileDirectoryHash', None)
```

Comment:
@c-o-l-o-r PLEASE, please, please snake case :)

Reply:
you got it captain